charts/localstack/values.yaml

```diff
@@ -41,6 +41,14 @@ debug: false
 # lambdaExecutor: ""
 # dataDir: ""
 
+# This will enable the Docker daemon binding and allow
+# Localstack to provide Lambdas and other AWS services
+# who got container runtime dependencies
+mountDind:
+  enabled: true
```

Comment:
Thinking more about this - I think we probably want to make this the default case, i.e., enable the `dind` sidecar container by default. Otherwise, key features like the Lambda API will not be fully functional (which should clearly work out of the box).

I'd suggest that we change this to the following (and make the corresponding changes in the chart templates above):
```
mountDind:
  disabled: false
  ...
```
Thoughts?

Reply:
i thought the same ^^ but realized that now that dockershim is deprecated, people are moving to `containerd` or `cri-o` so this won't work :( 

maybe we can enable it by default when workarounds are present for a working DinD with containerd/cri-o. WDYT?

Reply:
actually you are right ^^ DinD in that way we implement in here, works even with `containterd` as it does not rely on host machine docker daemon :rocket: lets revisit this on next PR